src/components/Ticket/Ticket.tsx

```diff
@@ -13,7 +13,7 @@ import { Download } from '@mui/icons-material';
 export const Ticket = ({ data }: { data: ITicket }) => {
```

Comment:
In desktop  and tab views, the seat layout appears at the top of the screen. May center that.

Reply:
done